packages/mantine/src/components/table/Table.tsx

```diff
@@ -128,7 +130,12 @@ export const Table: TableType = <T,>({
         return (
             <Fragment key={row.id}>
                 <tr
-                    onClick={() => row.toggleSelected()}
+                    ref={rowRef}
+                    onClick={() =>
+                        rowRef.current.querySelectorAll('.mantine-Table-root input[type=checkbox]:disabled').length > 0
+                            ? undefined
+                            : row.toggleSelected()
+                    }
```

Comment:
Row selection should still be enabled even when inside a disabled fieldset in some cases. Some row actions are still valid even in read-only mode. 

For example, we often have a table row action that opens another panel to view the details of the resource represented by the row. Even if the table is disabled you still want to be able to view the resource's details.

Reply:
I see! From the examples in [tanstack](https://tanstack.com/table/v8/docs/api/features/row-selection), I seem to have understood that row selection was for selecting and toggling the checkboxes associated to a row.

I'm not sure I understand the example completely. Are these details hidden inside a collapsible? Something like in the Sources panel?
<img width="1511" alt="image" src="https://github.com/coveo/plasma/assets/105073504/bb71a95c-da97-4804-bf7f-7dab07a16fd8">
If that's the case, if I'm not mistaken, this content becomes visible only when the user clicks on the arrow button and not on the row itself. The change here shouldn't affect access to the resource's details.

Reply:
You have an example of what I am referring to in your screenshot. In the table actions, we can see "Rescan", "Edit", "Activity", "... More". Those buttons still need to be activated even inside a disabled `fieldset`. Granted, not all of them, but the "Activity" one does, and the "... More" as well.